.ci/install_kubernetes.sh

```diff
@@ -11,22 +11,69 @@ set -o pipefail
 
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 
 echo "Install Kubernetes components"
 
 cidir=$(dirname "$0")
 source /etc/os-release || source /usr/lib/os-release
 kubernetes_version=$(get_version "externals.kubernetes.version")
 
-if [ "$ID" != "ubuntu" ]; then
-        echo "Currently this script only works for Ubuntu. Skipped Kubernetes Setup"
+if [ "$ID" != "ubuntu" ] && [ "$ID" != "centos" ]; then
+        echo "Currently this script does not work on $ID. Skipped Kubernetes Setup"
```

Comment:
Not strictly part of this PR, but could you add a commit to make the `exit` below specify an explicit value? Currently, the exit value depends on the result of the last command executed (which is admittedly always going to be `true` but still, best to be explicit).